src/main/java/uk/gov/hmcts/reform/managecase/service/noc/ApplyNoCDecisionService.java

```diff
@@ -120,14 +121,17 @@ private void applyDecision(CaseDetails caseDetails, JsonNode changeOrganisationR
         JsonNode organisationToRemoveNode = changeOrganisationRequestField.get(ORGANISATION_TO_REMOVE);
         Organisation organisationToAdd = objectMapper.convertValue(organisationToAddNode, Organisation.class);
         Organisation organisationToRemove = objectMapper.convertValue(organisationToRemoveNode, Organisation.class);
+        JsonNode createdBy = changeOrganisationRequestField.get(CREATED_BY);
 
         List<CaseUserRole> existingCaseAssignments = dataStoreRepository
             .getCaseAssignments(singletonList(caseDetails.getId()), null);
 
         if (organisationToAdd == null || isNullOrEmpty(organisationToAdd.getOrganisationID())) {
+            ((ObjectNode) orgPolicyNode).putIfAbsent("LastNoCRequestedBy", null);
```

Comment:
This should probably sit in the `applyRemoveRepresentationDecision()` method

Reply:
And can we check whether `putIfAbsent()` works in all scenarios - won't this only set to null if there's currently no value?
Might need another approach to always nullify it e.g.:
```suggestion
            ((ObjectNode) orgPolicyNode).set("LastNoCRequestedBy", objectMapper.nullNode());
```

Reply:
done